bin/live/pycbc_live_single_trigger_fits

```diff
@@ -176,7 +176,7 @@ if not os.path.exists(date_directory):
     raise FileNotFoundError(f"The directory {date_directory} does not exist.")
 
 files = [f for f in os.listdir(date_directory)
-         if args.file_identifier in f]
+         if args.file_identifier in f and f.endswith('hdf')]
```

Comment:
Some safety against files called e.g.: `.H1L1V1-Live-1252782892.07-8.hdf.Oo5mdy`

Reply:
This is good. Those are probably temporary files being written and being renamed with the final file name after the files have been properly written and closed.

Reply:
Which makes me wonder, do you do anything important with the `counter` variable below? It will count files even if they are not the files you are looking for.

Reply:
This is basically to make it so that progress is reported each 1000 files [check here](https://github.com/gwastro/pycbc/blob/master/bin/live/pycbc_live_single_trigger_fits#L186), so that we still get progress reports with verbose, but without spamming stderr